orchestration/scripts/generate-staging-table.sh

```diff
@@ -0,0 +1,51 @@
+set -euo pipefail
+
+# Copy-pasted from StackOverflow.
+function join_by { local d=$1; shift; echo -n "$1"; shift; printf "%s" "${@/#/$d}"; }
+
+# Point to BQ metadata we expect to be present on disk.
+declare -r TABLE_DIR=/bq-metadata/${TABLE}
+declare -r PK_COLS=$(cat ${TABLE_DIR}/primary-keys)
+
+# Build the WHERE clause of the SQL query.
+declare -a COMPARISONS=()
+declare -a DATAREPO_COLUMNS=(datarepo_row_id)
+# what are expected values of query type
+# use that marker to
+for c in ${PK_COLS//,/ }; do
+  COMPARISONS+=("${c} IS NOT NULL")
+  DATAREPO_COLUMNS+=("datarepo_${c}")
+done
+# rows to be updated
+if [[ ${QUERY_TYPE} = 'update'  ]] ; then
+COMPARISONS+=("datarepo_row_id IS NOT NULL")
+# new rows
+else
+COMPARISONS+=("datarepo_row_id IS NULL")
+fi
+declare -r FULL_DIFF=$(join_by ' AND ' "${COMPARISONS[@]}")
+declare -r REPO_KEYS=$(join_by ', ' "${DATAREPO_COLUMNS[@]}")
+
+declare -r TARGET_TABLE=${TABLE}_values
```

Comment:
I think this is going to collide with the existing new-values table for Jade.

The name used here should:
1. Probably mention `clingen` in the name, since it's for them
2. Be parameterized by the `QUERY_TYPE` variable.

If the two values of `query-type` we're going to use are `new_rows` and `updates`, we could have this be:
```
${TABLE}_clingen_${QUERY_TYPE}
```

Reply:
Ah okay - I like your suggested name for the TARGET_TABLE.
I'll change the query_type passed to "new_rows" and "updates"